x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -89,15 +89,13 @@ export class TaskManagerPlugin
       this.config!
     );
 
-    core.getStartServices().then(async () => {
```

Comment:
Not sure if this was necessary, everything _seems_ to work without it. If it is needed, we may need to change this to use a BehaviorSubject similar to how I did the changes in `alerts`.

cc @elastic/kibana-alerting-services

Reply:
Hmm, I wonder if we had anything before and then a cleanup made this redundant. @elastic/kibana-alerting-services?

Reply:
Yes, the changes provided in this PR is the right behavior. I think code here seems like was inherited (copied) from the alerts plugin, where start plugins were needed to the the task_manager.